serenity-core/src/test/groovy/net/thucydides/core/webdriver/chrome/WhenPassingChromeExperimentalOptionsToWebdriver.groovy

```diff
@@ -0,0 +1,34 @@
+package net.thucydides.core.webdriver.chrome
+
+import net.serenitybdd.core.webdriver.driverproviders.ChromeDriverCapabilities
+import net.thucydides.core.util.MockEnvironmentVariables
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class WhenPassingChromeExperimentalOptionsToWebdriver extends Specification {
+
+    @Unroll
+    def "Chrome experimental options should be added to chrome Capabilities"() {
+        given:
+        System.setProperty(optionText, optionValue.toString());
```

Comment:
Not a good idea to set the system properties directly in a unit test, as it can affect other tests as well.